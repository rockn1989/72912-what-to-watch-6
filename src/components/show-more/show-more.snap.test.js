import React from 'react';
import {render} from '@testing-library/react';
import ShowMore from './show-more';

it(`Should Review render correctly`, () => {
  const {container} = render(<ShowMore onShowMoreClick={jest.fn()} />);
  expect(container).toMatchSnapshot();
});
