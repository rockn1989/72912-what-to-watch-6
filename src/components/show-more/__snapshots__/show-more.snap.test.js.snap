// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Review render correctly 1`] = `
<div>
  <button
    class="catalog__button"
    type="button"
  >
    Show more
  </button>
</div>
`;
